queries.js

```diff
@@ -117,12 +117,65 @@ const deleteBook = (req, res) => {
     const id = req.params.id;
     pool.query('DELETE FROM books WHERE id = $1', [id], (err, result) => {
         if(err) {
-            throw error;
+            throw err;
         }
         res.status(200).send(`Delete a book with ID: ${id}`);
     })
 }
 
+//Get the list of category
+const getCate = (req, res) => {
```

Comment:
Should use entity's full name to make it clear for reader. Do the same for all variables below.

Reply:
ok